src/lib/Type.hs

```diff
@@ -17,7 +17,7 @@ module Type (
   oneEffect, lamExprTy, isData, asFirstOrderFunction, asFFIFunType,
   isSingletonType, singletonTypeVal, asNaryPiType,
   numNaryPiArgs, naryLamExprType, getMethodIndex,
-  extendEffect, effectsE, exprEffects, declNestEffects,
```

Comment:
This is newly exported, but I can't see any uses

Reply:
Good catch.  Unexported.